app/spell_check.py

```diff
@@ -20,6 +32,24 @@ def check_word(word_list):
 
     return results
 
+def check_other_lang(word_list, dictionary):
+    results = []
+    for idx, word in enumerate(word_list):
+        lower_word = str(unicodedata.normalize('NFC', word)).lower()
+        lower_word = lower_word.strip()
```

Comment:
If the word list was originally split at white space, then why does the word need to be stripped of leading and trailing whitespace here? Or is the unwanted whitespace being introduced somewhere along the way after the initial splitting?

Reply:
This actually seems to be a leftover artefact from before I built the parse_txt_other_lang function. When it was passing through the English parse function whitespace was appearing. Ive just tested it without stripping and it is still working properly, so I can either remove it or leave it in for redundancy, just let me know what you prefer

Reply:
I think we should definitely remove it if it's no longer needed. Redundant code makes it harder to read and understand down the road.

Reply:
will do, ill take care of it later today

Reply:
Removed

Reply:
Also taken care of by regex pr